### ISSUE TITLE
UnrealLink no longer updates

### ISSUE THREAD

**Issue by Snipe3000** [JetBrains/UnrealLink#257]
Whenever I try to update the plugin I get this error:

The IDE failed to install or update some plugins.
Please try again, and if the problem persists, please report it
to https://jb.gg/ide/critical-startup-errors

The cause: java.nio.file.AccessDeniedException: C:\Users\Me\AppData\Roaming\JetBrains\Rider2023.1\plugins\UnrealLink\dotnet\RiderPlugin.UnrealLink.dll

**Comment by DecoyRS** [comment]
Hello @Snipe3000
Are there any other messages in the "Install RiderLink" panel? This looks like you're trying tu update plugin while Unreal Editor or Commandlet is running. Can you make sure that none of them is running and try installing it once again?

**Comment by Snipe3000** [comment]
Yeah that works. I have to shut everything down to get it to update
